diff --git a/node_modules/squad-rcon/lib/index.js b/node_modules/squad-rcon/lib/index.js
index f30e883..69a8b97 100644
--- a/node_modules/squad-rcon/lib/index.js
+++ b/node_modules/squad-rcon/lib/index.js
@@ -408,7 +408,7 @@ class Rcon extends EventEmitter {
         this.client = net.createConnection({
             host: this.host,
             port: this.port,
-            noDelay: true,
+            noDelay: false,
         });
         this.logger.log('Connecting');
         this.client.on('data', (data) => {
